### PR TITLE
Combined temp code with SD code

### PR DESCRIPTION
**Pull request by skolodny-leffell**

### CODE REVIEW

File: main/main.ino [hunk 1]
```diff
@@ -1,9 +1,22 @@
 #include <SD.h>
 #include <SPI.h>
 #include <DHT22.h>
-//define pin data
+#include <OneWire.h>
```
Comment (arimessinger) [inline main/main.ino:4]: 4-9 ✅

File: main/main.ino [hunk 2]
```diff
@@ -1,9 +1,22 @@
...
+#include <DallasTemperature.h>
+
+// Data wire is conncted to the Arduino digital pin 4
+#define INSIDE 4
+#define OUTSIDE 2
 #define pinDATA 7 // SDA, or almost any other I/O pin
 
+// Setup a oneWire instance to communicate with any OneWire devices
```
Comment (arimessinger) [inline main/main.ino:12]: 12-19 ✅

**Review by arimessinger — COMMENTED** [review]
this is a test

**Comment by arimessinger** [comment]
> this is a test

resolved

**Review by arimessinger — CHANGES_REQUESTED** [review]
Test and make sure that the code works and sends to the sd card in a readable file format (.csv would be great)

**Review by opodietz — APPROVED** [review]

**Review by arimessinger — APPROVED** [review]
Looks good to go. Merge with main!

**Pull request merged**